### ISSUE TITLE
Suggestion: Blog announcing GSOC 2023 interns

### ISSUE THREAD

**Issue by reshamas** [pymc-devs/pymc.io#87]
Are people interested in doing a blog announcing the Google Summer of Code Students for 2023, like we did last year?
https://www.pymc.io/blog/blog_gsoc_2022.html

We could update the title of the blog, such as 2023 Interns (instead of GSOC Summer 2023 Students), so:

**Meet our 2023 PyMC Interns**

**Comment by error9098x** [comment]
Sure, I can do this !

**Comment by error9098x** [comment]
Where can I get the details + photos of the interns ?

**Comment by reshamas** [comment]
@error9098x 
Are you still available to do this?
We have the info in a Google doc to submit to a PR.

For reference, the 2022 blog file:
https://github.com/pymc-devs/pymc.io/blob/main/blog/blog_gsoc_2022.md

We would need to create a new file:  blog_interns_2023.md

**Comment by error9098x** [comment]
Yeah, sure I am available, I will do this task just provide me the Google
docs :)

**Comment by reshamas** [comment]
@error9098x 
Here is the link:
https://docs.google.com/document/d/19BMX5JI06BI96747N2Wx1S6MDbFh0PtwYBCQQs0rCew/edit?usp=sharing

**Comment by error9098x** [comment]
> @error9098x Here is the link: https://docs.google.com/document/d/19BMX5JI06BI96747N2Wx1S6MDbFh0PtwYBCQQs0rCew/edit?usp=sharing

Hey, could you please check the pull request, thanks
